h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -978,6 +977,14 @@ public void cv_mainModelScores(int N, ModelMetrics.MetricBuilder mbs[], ModelBui
     // Now, the main model is complete (has cv metrics)
     DKV.put(mainModel);
   }
+  
+  public void cv_reduceCVBuilders(ModelMetrics.MetricBuilder mbsMain, ModelMetrics.MetricBuilder mbsReduced){
```

Comment:
Do you need this method? It just delegates to "reduceForCV"? Instead of calling this on line https://github.com/h2oai/h2o-3/pull/5961/files#diff-892e6d71b6905c34c9126b46bf544491e10b6c6394a83b58de5a2ead0eca6ca7R908 - can you directly call reduceForCV? It seems a bit more cleaner to make the method "reduceForCV" part of the interface of MetricBuilder rather than ModelBuilder.

Reply:
It is overridden in ClusteringModelBuilder.java. The implement reduceForCV in MetricBuilder is possible, but this way made more sense to me. Thanks for the suggestion.

Reply:
yes, I know it is overriding but it is only doing that to delegate to some other method - which is what I do not understand

if it was definited on metric builder there would be no delegating necessary